conda/cli/conda_argparse.py

```diff
@@ -85,6 +85,12 @@ def generate_parser():
         action="store_true",
         help=SUPPRESS,
     )
+    p.add_argument(
+        "-P",
```

Comment:
This might be confusing with the lower-case `-p` for `--path`?
```suggestion
```

Reply:
Fair. Do you have better suggestion? 😁 We could also just not even include an alias for it.

Reply:
I was thinking about not even including it originally.

Reply:
Yeah agreed, let's just keep `--no-plugins`, that's descriptive and explicit.

Should we ALSO add a config option BTW?